scripts/HPC_chatbot_preprocessor/parsed_mds/generic/account/account_paragraph_3.txt

```diff
@@ -0,0 +1,15 @@
+How do SSH keys work?
+-   an SSH public/private key pair can be seen as a lock and a key
+-   the SSH public key is equivalent with a lock: you give it to the
+    VSC and they put it on the door that gives access to your account.
+-   the SSH private key is like a physical key: you don't hand it out
+    to other people.
+-   anyone who has the key (and the optional password) can unlock the
+    door and log in to the account.
+-   the door to your VSC account is special: it can have multiple
+    locks (SSH public keys) attached to it, and you only need to open
+    one lock with the corresponding key (SSH private key) to open
+    the door (log in to the account).
```

Comment:
i would add a pargrpah to explain what an optional but recommend password means. it's like a safe at home where you keep the key to open the door from work